addon/session-stores/cookie.js

```diff
@@ -189,7 +189,7 @@ export default BaseStore.extend({
       document.cookie = `${this._oldCookieName}=`;
     }
 
-    if (isBlank(value)) {
+    if (isBlank(value) && expiration !== 0) {
```

Comment:
This is for detecting a cookie clear which runs `this._write(null, 0)`. What would be a better way of handling this?